packages/next/package.json

```diff
@@ -69,7 +69,7 @@
     "@babel/plugin-syntax-dynamic-import": "7.2.0",
     "@babel/plugin-transform-modules-commonjs": "7.7.0",
     "@babel/plugin-transform-runtime": "7.6.2",
-    "@babel/preset-env": "7.7.1",
+    "@babel/preset-env": "7.9.5",
```

Comment:
Should we update `@babel/core` to the same version?

Reply:
Yeah let's update the babel deps